__tests__/UI-tests/Forgot-Password-test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { render } from "@testing-library/react-native";
-import ForgotPasswordScreen from "../app/screens/auth/forgot_password_screen";
+import ForgotPasswordScreen from "../../app/screens/auth/forgot_password_screen";
```

Comment:
Can we start using the @/app/screen ... syntax it will help avoid errors with path imports